cmd/login.go

```diff
@@ -81,7 +81,7 @@ var loginCmd = &cobra.Command{
 
 func init() {
 	RootCmd.AddCommand(loginCmd)
-	loginCmd.Flags().StringVarP(&controllerURL, "controllerURL", "s", "http://localhost:8080", "Controllers URL")
+	loginCmd.Flags().StringVarP(&controllerURL, "controllerURL", "s", "http://localhost:80", "Controllers URL")
```

Comment:
I don't think that using :80 is a good Idea, also, if we were to use it, you shouldn't need to specify it then